filebeat/tests/system/test_reload_modules.py

```diff
@@ -170,6 +170,7 @@ def test_load_configs(self):
         self.render_config_template(
             reload_path=self.working_dir + "/configs/*.yml",
             reload_type="modules",
+            reload=True,
```

Comment:
Without explicit `reload` to true (though the path exists), this test should fail. Let me know if this could be the expected/ideal behaviour. If not we could do something like, if `reload_path`exists, assume `reload` is enabled.